fileserver.gr

```diff
@@ -45,14 +47,13 @@ let serve = (path) => {
 }
 
 let guestpath = (env) => {
+
+    // Backward compat for an older version of Wagi that had PATH_INFO wrong.
+    // X_RELATIVE_PATH was removed before Wagi 0.4
     match (Map.get("X_RELATIVE_PATH", env)) {
```

Comment:
Doesnt X_RELATIVE_PATH need prefixing with a / to make this work? Or did I have my implementation wrong in wagi-dotnet?

Reply:
Yes, I think you are right.